python/docs/source/getting_started/ps_install.rst

```diff
@@ -47,20 +47,20 @@ To put your self inside this environment run::
 
     conda activate koalas-dev-env
 
-The final step required is to install Koalas. This can be done with the
+The final step required is to install pandas APIs on Spark. This can be done with the
 following command::
 
     conda install -c conda-forge koalas
 
-To install a specific Koalas version::
+To install a specific pandas APIs on Spark version::
 
     conda install -c conda-forge koalas=1.3.0
 
 
 Installing from PyPI
 ~~~~~~~~~~~~~~~~~~~~
 
-Koalas can be installed via pip from
+Pandas APIs on Spark can be installed via pip from
```

Comment:
FYI: We will refine/remove the such an incorrect/obsolete pages or contents in the follow-up PRs.